docs/README.md

```diff
@@ -1,149 +1,77 @@
+
+OpenWhisk runs application logic in response to events or direct invocations from web or mobile apps over HTTP. Events can be provided from Bluemix services like Cloudant and from external sources. Developers can focus on writing application logic, and creating actions that are executed on demand. The benefits of this new paradigm are that you do not explicitly provision servers and worry about auto-scaling, or worry about high availability, updates, maintenance and pay for hours of processor time when your server is running but not serving requests. Your code executes whenever there is an HTTP call, database state change, or other type of event that triggers the execution of your code. You get billed by millisecond of execution time (rounded to the nearest 100ms in case of OpenWhisk) or on some platforms per request (not supported on OpenWhisk yet), not per hour of JVM regardless whether that VM was doing useful work or not.
+
+This programming model is a perfect match for microservices, mobile, IoT and many other apps – you get inherent auto-scaling and load balancing out of the box without having to manually configure clusters, load balancers, http plugins, etc. If you happen to run on IBM Bluemix, you also get a benefit of zero administration - meaning that all of the hardware, networking and software is maintaned by IBM. All you need to do is to provide the code you want to execute and give it to your cloud vendor. The rest is “magic”. Good introduction into the serverless programming model is available on [Martin Fowler's blog](https://martinfowler.com/articles/serverless.html).
+
+## Overview
+- [How OpenWhisk works](./about.md)
+- [Common uses cases for Serverless applications](./use_cases.md)
+- [Sample applications](./samples.md)
+
+<!--
+- [Is serverless a good fit for my application?](./goodfit.md)
+-->
+
+<!-- TODO - need to add the following items and pages in the future:
+- Concurrency
+- Error processing
+- Security
+- Scalability
+- Logging
+- Pricing (for Bluemix only)
+-->
+
+## Implementation guide
+- [Setting up and using OpenWhisk CLI](./cli.md)
+- [Using OpenWhisk from an iOS app](./mobile_sdk.md).
+- [Using REST APIs with OpenWhisk](./rest_api.md)
+
+<!-- TODO - need to add the following items and pages in the future:
+- Development workflow
+- IDE and toolchain
+- When OpenWhisk is a good choice and when it is not
+- Building applications from scratch
+- Extending existing applications
+- Using OpenWhisk as the integration tool
+- Integrating with API Management
+- Running OpenWhisk (PaaS versus RYO, private, public, dedicated)
+- Best practices for rules
+- Performance considerations
+- Deployment of applications
+- Debugging
+- Monitoring of applications
+- Writing applications with maximum portability
+- Publishing events
+- Out of the box services and triggers
+- CDN integration
+- Security
+- Limitations
+-->
+
+## Programming model
+- [System overview](./reference.md)
```

Comment:
i dont think so - the real system details are in the reference.md file. The about.md title needs to be changed - it is not a system overview.